### DISCUSSION TITLE
cloned and re-IP netbox server weird issue

### DISCUSSION THREAD

**Discussion by TalkBackJohnny** [netbox-community/netbox#18160] [Help Wanted!]
I was testing the viability of Netbox for our environment and found that it worked great. I had a bunch of real world information in my test Netbox that I didn't want to lose so I cloned the VM, renamed and re-IP'd into my production environment and went to work adding more info.

I just noticed today that when I go into the RACKS:RACKS:"PICK any rack": hover over any of the devices in the rack the "hover link" at the bottom left shows the IP of the old test server. Clicking on that device in the rack tries the old VM. I've powered off the testVM since I've found this issue.

Adding new device into the existing rack still points to the old server when hover/click, but it still adds it correctly to my production netbox.

any help would be appreciated.

**Comment by candlerb** [comment]
Netbox generates absolute hyperlinks in SVG images. The hostname it uses for those links comes from the `Host:` or `X-Forwarded-Host:` header of the incoming request.

You can check this using `tcpdump -i lo -s0 -A -nn tcp port 8001` on the Netbox server, while making an incoming request.

If that header is wrong, usually it's because of a problem with the configuration of the front-end proxy (nginx or apache). You can find working example configs [here](https://github.com/netbox-community/netbox/tree/master/contrib). e.g. if it's apache then you'll want `ProxyPreserveHost On`

I do suggest you switch off the old/test instance, in case you accidentally navigate to it.

Aside: if you want to clone the contents of one Netbox instance to another, an alternative to cloning the whole VM is to export the postgres database and any media/attachments. The process is documented under [Replicating Netbox](https://netboxlabs.com/docs/netbox/en/stable/administration/replicating-netbox/).
